pandas/core/frame.py

```diff
@@ -3780,7 +3780,7 @@ def nlargest(self, n, columns, keep='first'):
 
         Examples
         --------
-        >>> df = DataFrame({'a': [1, 10, 8, 11, -1],
+        >>> df = pd.DataFrame({'a': [1, 10, 8, 11, -1],
         ...                 'b': list('abdce'),
```

Comment:
Although this won't fail, but style-wise, notice how the "a", "b", and "c" rows are aligned and start at that same location o n their respective lines.  Let's keep it that way (i.e. adjust the "b" and "c" rows).

Reply:
Follow this comment for one of your other fixes down below (the "column=['a', 'b']").